docs/website/gatsby-config.js

```diff
@@ -130,9 +130,11 @@ module.exports = {
           },
           {
             resolve: 'gatsby-remark-autolink-headers',
-            options: {
-              offsetY: '64', // <600: 48; >600:64
-            },
+            // Oprions don't work with gatsby-plugin-mdx,
```

Comment:
1. Typo option
2. Rename to Option `offsetY`
3. Bug report URL on Gatsby website

Reply:
3. There is a link in the description of the pull request